test/test-functions

```diff
@@ -1336,6 +1355,12 @@ inst_libdir_file() {
     fi
 }
 
+setup_suse() {
+    ln -s ../usr/bin/systemctl $initdir/bin/systemctl
+    ln -s ../usr/lib/systemd $initdir/lib/systemd
+    inst_simple "/usr/lib/systemd/system/haveged.service"
```

Comment:
haveged is per default in the SUSE initrd as an entropy creator. Omitting it will fail the qemu test when the host initrd is used and systemd deserializes haveged.service after switching from initrd to the real root.